### ISSUE TITLE
Missing Function For Unity Events

### ISSUE THREAD

**Issue by Chifilly** [Unity-Technologies/InputSystem#791]
I'm unsure of the cause of this, but all of the events for my inputs say that the function is missing, even though the functions aren't missing, and when I enter play mode, the events fire exactly as they should.

The biggest issue I have now though is that I can't add new events, because none of my functions appear unless they have no parameters, but I need to be able to have access to `CallbackContext`.

This worked perfectly fine previously, and the function gets called perfectly fine in play mode, but when I try and select a function for the event, none of the functions show up at all.

Here's an example of a function, this is the basic skeleton of all my event functions, which is correct according to the documentation (and they work in-game);
```csharp
public void Move(InputAction.CallbackContext context)
{
    // ...
}
```

This is what it shows in the editor:
![unity_input_system_missing_functions](https://user-images.githubusercontent.com/3699889/63237102-79447f00-c238-11e9-97df-eb3915b9fee7.PNG)

I'm using version `preview-0.9.3` of the Input System package, and version `2019.3.0a12` of Unity. 

I was previously using `preview-0.9.0` of the Input System package and it still had the issue (I updated to see if it would fix it). I'm unsure if the issue was in `2019.3.0a11` of Unity, because I didn't need to change the events when I was on that version, so I never checked.

EDIT:
I got my project open in 2019.2.1f1 and it worked correctly. I can't test it on 2019.3.0a11 because I uninstalled it and can't download it via Unity Hub anymore and I can't find an external download. But it seems that the issue occurred when upgrading to one of the newer alpha versions of Unity.

**Comment by Rene-Damm** [comment]
That's an odd one. I'd gut-blame it on the alpha. The alphas can have rather wonky issues and that one looks like something went quite wrong there with the scripting internals and UnityEvent. Doesn't seem like an issue on the side of the input system.

>I got my project open in 2019.2.1f1 and it worked correctly.

Ok, thanks for posting an update.

Let's see if the issue comes back when 2019.3 is getting closer to final.

**Comment by Chifilly** [comment]
For anyone that also has this issue, it is now on the Unity bug tracker after I reported it with an example and they were able to reproduce it. View the issue [here](https://issuetracker.unity3d.com/issues/events-generated-by-the-player-input-component-do-not-have-callbackcontext-set-as-their-parameter-type).

**Comment by Chifilly** [comment]
For anyone with this issue, it is fixed in 2019.3.0b4
